lib/features/support/hooks.rb

```diff
@@ -64,8 +65,10 @@
 
       config.capabilities['app'] = "storage:#{config.app}"
     else
+      $logger.error 'Browser support with Sauce Labs not yet implemented'
+      exit(1)
```

Comment:
Does this need to exit or can it raise instead?

Reply:
Changed to raise.